src/app/shared/image/image-loader.component.ts

```diff
@@ -4,18 +4,22 @@ import { Component, Input, ElementRef } from '@angular/core';
   selector: 'image-loader',
   styleUrls: ['image-loader.component.css'],
   template: `
-    <img *ngIf="src" [src]="src" class="loading" (load)="onLoad()"/>
+    <img *ngIf="src" [src]="src" class="loading" (load)="setBackgroundImage(src)" (error)="setBackgroundImage(placeholderError)"/>
+    <img *ngIf="!src" [src]="placeholderImage" class="loading" (load)="setBackgroundImage(placeholderImage)" (error)="setBackgroundImage(placeholderError)"/>
```

Comment:
Also ... even though I was the one to add that `loading` class, I'm pretty sure it does nothing.